web/main/templates/includes/casebook_search.html

```diff
@@ -0,0 +1,50 @@
+{% load current_query_string %}
+
+<section class="casebook-search">
+  <header class="advanced-search view-searches-show">
+    <form novalidate="novalidate" class="simple_form search" accept-charset="UTF-8">
+      {# Hidden field to preserve active tab's category during form submission #}
+      {# Note that 'category' is singular, but 'type' is plural #}
```

Comment:
This category/type comment is no longer accurate, right, since your recent PR?

Reply:
Ah yes, good catch, this is a holdover.